README.md

```diff
@@ -51,13 +51,13 @@ Use the following resources to help you find the Active Record query methods tha
 
 You'll be defining the following methods:
 
-* `#highest_rating`: this method should return the highest value in the ratings column. *hint*: if there is a `#minimum` Active Record method, might there be a `#maximum` method?
-* `#most_popular_show`: this method should return the show with the highest rating. *hint*: use the `#highest_rating` method as a helper method.
-* `#lowest_rating`: returns the lowest value in the ratings column.
-* `#least_popular_show`: returns the show with the lowest rating.
-* `#ratings_sum`: returns the sum of all of the ratings.
-* `#popular_shows`: returns an array of all of the shows that have a rating greater than `5`. *hint*: use the `#where` Active Record method.
-* `#shows_by_alphabetical_order`: returns an array of all of the shows sorted by alphabetical order according to their names. *hint*: use the `#order` Active Record method.
+* `::highest_rating`: this method should return the highest value in the ratings column. *hint*: if there is a `#minimum` Active Record method, might there be a `#maximum` method?
```

Comment:
Hey @kwebster2 !

I think the curriculum most often uses `.method_name` to denote a class method — mind flipping it over to that?

I hear you on following the Ruby docs standard, but I think `::` would be unfamiliar to most students.